### PR TITLE
refactor([t-z]*): apply naming guidelines where API is as desired

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

File: completions/vncviewer [hunk 1]
```diff
@@ -13,11 +13,11 @@ _vncviewer_bootstrap()
...
     complete -F $fname vncviewer
     $fname "$@" # Generate completions once for now
-    unset -f _vncviewer_bootstrap
+    unset -f "$FUNCNAME"
 } &&
-    complete -F _vncviewer_bootstrap vncviewer
+    complete -F _comp_cmd_vncviewer__bootstrap vncviewer
 
-_tightvncviewer()
+_comp_cmd_tightvncviewer()
```
Comment (akinomyoga) [inline completions/vncviewer:20]: https://github.com/scop/bash-completion/blob/3ed0e1f9bed7b7ec7950a20a60ac9345aab8f7a3/completions/vncviewer#L8-L9

These also need to be updated.

File: completions/vncviewer [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # bash completion for vncviewer                            -*- shell-script -*-
 
-_vncviewer_bootstrap()
+_comp_cmd_vncviewer__bootstrap()
```
Comment (akinomyoga) [inline completions/vncviewer:3]: https://github.com/scop/bash-completion/blob/3ed0e1f9bed7b7ec7950a20a60ac9345aab8f7a3/test/t/test_vncviewer.py#L4

File: completions/tar [hunk 3]
```diff
@@ -710,22 +710,22 @@ _posix_tar()
...
 
-    __tar_try_list_archive && return
+    _comp_cmd_tar__try_list_archive && return
 
     # file completion on relevant files
     _filedir
 }
 
-_tar()
+_comp_cmd_tar()
```
Comment (akinomyoga) [inline completions/tar:719]: https://github.com/scop/bash-completion/blob/3ed0e1f9bed7b7ec7950a20a60ac9345aab8f7a3/test/t/test_tar.py#L8

File: completions/vncviewer (hunk 1 above)
Comment (scop) [reply completions/vncviewer:20]: Thanks! Addressed in a fixup commit (gitlint will fail with that until #940 is merged). Ditto the two test ignores below.

**Review by akinomyoga — APPROVED** [review]

**Pull request merged**